### PR TITLE
Allow verify (cacert location) to be specified in ServiceRestProxy options.

### PR DESCRIPTION
**Pull request by jondmcelroy**
Updated readme to document changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -154,12 +154,20 @@ To use proxies during HTTP requests, set system variable `HTTP_PROXY` and the pr
...
-1. Download the cacert.pem file from [cURL site](http://curl.haxx.se/docs/caextract.html).
-2. Open your php.ini file and add the following line:
-
-```ini
-curl.cainfo = "<absolute path to cacert.pem>"
-```
+1. Download the cacert.pem file from [cURL site](http://curl.haxx.se/docs/caextract.html). 
+
+2. Then either:
+	* Open your php.ini file and add the following line:
```
Comment (XiaoningLiu) [inline README.md]: Keep padding align with below

File: README.md [hunk 2]
```diff
@@ -154,12 +154,20 @@ To use proxies during HTTP requests, set system variable `HTTP_PROXY` and the pr
...
+		```ini
+		curl.cainfo = "<absolute path to cacert.pem>"
+		```
+		OR
+    * Point to the cacert in the options when creating the Relevant Proxy.
```
Comment (XiaoningLiu) [inline README.md]: Keep padding align with above

File: README.md [hunk 3]
```diff
@@ -154,12 +154,20 @@ To use proxies during HTTP requests, set system variable `HTTP_PROXY` and the pr
...
+		```php
+		//example of creating the FileRestProxy
+		$options["http"] = ["verify" => <absolute path to cacert.pem>];
```
Comment (XiaoningLiu) [inline README.md]: Add quotes like "verify" => "<absolute path to cacert.pem>"

**Comment by jondmcelroy** [comment]
Updated whitespace and cleaned up unnecessary comments.

**Comment by katmsft** [comment]
Hi @jondmcelroy , thank you for your contribution. However I find no test cases related to the change. Can you add one?
I'm fine with putting the cert somewhere in the test folder and test with it. Any thoughts?

**Comment by jondmcelroy** [comment]
Hi @katmsft,
Where you thinking a functional test to see if it can connect or just a test to see if the cert is set in the client?

**Comment by katmsft** [comment]
The added cases are fine with me. Thanks for the quick response.

**Pull request merged**